### PR TITLE
[feat]: [PIE-6231]: Renaming pipeline crud methods to proper operation names

### PR DESCRIPTION
**Pull request by archit-harness**
## What is the issue?
Renaming pipeline crud operations to proper operation name, as validate method is not inherited from current methods name

What is the change?
Rename pipeline crud methods

Impact?
No impact

Testing? 
a) If Manual -> Run basic flows of pipeline crud
b) If indexes -> No
c) If particular/complete sanity ran -> No

ReleaseNotes summary needed? No

1. Is it new task type? No
2. Did you make changes to the code in a backward-compatible manner? no 
3. Did you add a new class which is added in kryo serialization? - No
4. Will the new version of the delegate receive old version task and process? No
5. If new version of the pipeline-service receives response for the old task will it be able to handle it successfully? No
6. Did you make a change in delegate task which will be created in pipeline-services via grpc call to manager? No 
7. If the older version of pipeline-service creates the task parameter and calls newer cg manager, will it be okay? No
8. if the newer version of pipeline-service creates the task parameter and calls older cg manager, will it be okay? No
9. Did you remove an enum → No
10. Did you add an enum(That too at end of list) → No
11. FeatureFlag Added/Removed -> No
12. Verify Changes with affects any -> No

## Checklist
- [x] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [x] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>
  
- Feature build: `trigger feature-build`
- Immutable delegate `trigger publish-delegate`
</details>

<details>
  <summary>PR Check triggers</summary>
  
  You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ti0, ti1`
  
- Compile: `trigger compile`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- Feature Name Check: `trigger featurenamecheck`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/39929)

### CODE REVIEW

**Review by naidu-harness — CHANGES_REQUESTED** [review]

File: pipeline-service/service/src/main/java/io/harness/pms/plan/execution/handlers/ExecutionInfoUpdateEventHandler.java [hunk 1]
```diff
@@ -63,7 +63,8 @@ void updateExecutionInfoInPipelineEntity(Ambiance ambiance, PlanExecution planEx
       String orgId = AmbianceUtils.getOrgIdentifier(ambiance);
       String projectId = AmbianceUtils.getProjectIdentifier(ambiance);
       String pipelineId = ambiance.getMetadata().getPipelineIdentifier();
-      Optional<PipelineEntity> pipelineEntity = pmsPipelineService.get(accountId, orgId, projectId, pipelineId, false);
+      Optional<PipelineEntity> pipelineEntity =
+          pmsPipelineService.getAndValidatePipeline(accountId, orgId, projectId, pipelineId, false);
```
[the remark this answers is not in the document]
Comment (archit-harness) [reply pipeline-service/service/src/main/java/io/harness/pms/plan/execution/handlers/ExecutionInfoUpdateEventHandler.java:67]: Will be handled in next PR, not creating big changes in single PR

File: pipeline-service/service/src/main/java/io/harness/pms/plan/execution/handlers/ExecutionSummaryCreateEventHandler.java [hunk 2]
```diff
@@ -99,7 +99,8 @@ public void onStart(OrchestrationStartInfo orchestrationStartInfo) {
 
     ExecutionMetadata metadata = planExecution.getMetadata();
     String pipelineId = metadata.getPipelineIdentifier();
-    Optional<PipelineEntity> pipelineEntity = pmsPipelineService.get(accountId, orgId, projectId, pipelineId, false);
+    Optional<PipelineEntity> pipelineEntity =
+        pmsPipelineService.getAndValidatePipeline(accountId, orgId, projectId, pipelineId, false);
```
[the remark this answers is not in the document]
Comment (archit-harness) [reply pipeline-service/service/src/main/java/io/harness/pms/plan/execution/handlers/ExecutionSummaryCreateEventHandler.java:103]: Will be handled in next PR, not creating big changes in single PR

File: pipeline-service/service/src/main/java/io/harness/pms/resourceconstraints/service/PMSResourceConstraintServiceImpl.java [hunk 3]
```diff
@@ -151,7 +151,8 @@ private PipelineEntity getPipelineEntity(
              new AutoLogContext(ImmutableMap.of("accountId", accountId, "orgIdentifier", orgIdentifier,
                                     "projectIdentifier", projectIdentifier, "pipelineIdentifier", pipelineIdentifier),
                  OVERRIDE_ERROR)) {
-      return pipelineService.get(accountId, orgIdentifier, projectIdentifier, pipelineIdentifier, false)
+      return pipelineService
+          .getAndValidatePipeline(accountId, orgIdentifier, projectIdentifier, pipelineIdentifier, false)
```
[the remark this answers is not in the document]
Comment (archit-harness) [reply pipeline-service/service/src/main/java/io/harness/pms/resourceconstraints/service/PMSResourceConstraintServiceImpl.java:155]: Will be handled in next PR, not creating big changes in single PR

**Review by naidu-harness — APPROVED** [review]

**Comment by archit-harness** [comment]
trigger messagecheck

**Review by sahilhindwani-harness — APPROVED** [review]
> Reviewed 36 of 43 files at r1, 8 of 8 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/39929)* status: 44 file(s) to review. 3. unresolved discussions.

**Pull request merged**